src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc

```diff
@@ -1,96 +0,0 @@
----
```

Comment:
@MichalMaler, you deleted this file (or introduced `assembly_configuring-a-workspace-using-a-devfile.adoc` in its stead), but this deleted file is now not available -- but it's still being included (from `assembly_using-developer-environments-workspaces.adoc`). Please, remember to always build your PRs. You would see the following error message:
```
asciidoctor: ERROR: assembly_using-developer-environments-workspaces.adoc: line 18: include file not found: /che-docs/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc
```